addons/udes_delivery_control/views/stock_picking_type.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_type_form_delivery_control_inherit" model="ir.ui.view">
+        <field name="name">stock.picking.type.form.delivery.control.inherit</field>
+        <field name="model">stock.picking.type</field>
+        <field name="inherit_id" ref="udes_stock.view_picking_type_form_inherit"/>
+        <field name="arch" type="xml">
+            <field name="u_max_reservable_pallets" position="after">
+                <field name="u_is_delivery_control"/>
```

Comment:
Nit: I think we can remove this view override to be honest. `u_is_delivery_control` isn't a config option, it's something that should only be hardcoded by us. And we'll know if we're looking at the Delivery Control operation type anyway.

For reference the Advanced Stock Move (`udes_stock_move`) doesn't show the `u_is_stock_move` in the Picking Type view.